plugins/woocommerce/src/Admin/WCAdminHelper.php

```diff
@@ -149,7 +149,7 @@ public static function is_current_page_store_page() {
 		$store_pages = apply_filters( 'woocommerce_store_pages', $store_pages );
 
 		foreach ( $store_pages as $page_slug => $page_id ) {
-			if ( is_page( $page_slug ) || ( $page_id > 0 && is_page( $page_id ) ) ) {
+			if ( $page_id > 0 && is_page( $page_id ) ) {
```

Comment:
Lets leave this as is, as it fixes the issue and I don't want to accidently introduce something new.
But it does look like instead of doing a foreach we could of also done:
`if ( is_page( array_values( $store_pages ) ) { return true; }` as `is_page` also supports an array: https://developer.wordpress.org/reference/functions/is_page/